source/index.blade.php

```diff
@@ -106,7 +107,86 @@ class="shape shape-2"
       </div>
     </section>
     <!-- ====== About End ====== -->
+    <section>
+    <div class="_carousel">
+      <div class="slide-track">
+        
+            <div class="slide">
+              <a href="https://www.youtube.com/watch?v=3Z780EOzIQs" target="_blank">
+                  <img src="{{ $page->baseUrl }}assets/images/companies/empresa1.png" alt="" class="Oi">
+              </a>
+            </div>
+            
+            <div class="slide">
+                <a href="https://www.youtube.com/watch?v=3Z780EOzIQs" target="_blank">
+                  <img src="{{ $page->baseUrl }}assets/images/companies/empresa2.png" alt="">
```

Comment:
I would expect the `alt` text of each logo to be the companies name.

For example, this one would be `alt="Amazon"`.

Reply:
> If you're certain you're allowed to use these images in this way, I'd recommend using a tool like [REUSE by the FSFE](https://reuse.software/) for declaring the licenses properly.

Excellent point!

At LibreSign we are using a REUSE GitHub Action to validate this. It's a good idea to a new PR here implementing REUSE.

Reply:
> I would expect the alt text of each logo to be the companies name.

It's good to usability.

Reply:
@SethFalco I wrote a possible to-do list with steps to implement SPDX.